x-pack/plugins/triggers_actions_ui/public/application/sections/alert_details/components/alert_instances.tsx

```diff
@@ -135,22 +135,36 @@ export function AlertInstances({
   unmuteAlertInstance,
   requestRefresh,
 }: AlertInstancesProps) {
+  const [pagination, setPagination] = useState<Pagination>({ index: 0, size: 10 });
```

Comment:
there are a couple of `10` literals throughout the PR; should these be a constant defined somewhere?

Reply:
Well, only once in the plugin itself, the others are tests where the duplication makes sense to me. But that said, alert list has it too, I'll pull both into a constant.